### PR TITLE
feat: add external resource component

### PR DESCRIPTION
**Pull request by dylanzheng94**
## 🔗 Linked issue
 - #296 

## ❓ Type of change
- [x] 👌 Enhancement (improving an existing functionality like performance)
- [x] ✨ New feature (a non-breaking change that adds functionality)

Screenshot 2023-03-07 at 4 20 40 PM

## 📝 Checklist

- [x] I have linked an issue or discussion.

- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: app/utils/constants.js [hunk 1]
```diff
@@ -11,3 +13,11 @@ export const FR_LOCALE = 'fr-CA'
...
 export const EN = 'en'
 export const FR = 'fr'
+
+export const EXAMPLES_LINK_ID = 'ex'
+
+export const getExamplesLinkName = (currentLocale) => {
+  return currentLocale === EN_LOCALE
+    ? langEN.examples_from_our_work
+    : langFR.examples_from_our_work
+}
```
Comment (daine) [inline app/utils/constants.js:23]: Not a biggie, but this doesn't seem like the right place for this (file is constants)

File: app/utils/listItemsUtility.js [hunk 2]
```diff
@@ -35,3 +35,19 @@ export const generateResources = (resourceItems, locale) => {
...
+  return externalResourceItems
+    .filter(
+      (externalResourceItem) =>
+        externalResourceItem != null && externalResourceItem?.title != null
+    )
+    .map((externalResourceItem) => ({
+      resourceType: externalResourceItem.resourceType.name,
+      source: new URL(externalResourceItem.url).host.startsWith('www.')
+        ? new URL(externalResourceItem.url).host.substring(4)
+        : new URL(externalResourceItem.url).host,
```
Comment (daine) [inline app/utils/listItemsUtility.js:49]: Is this removing the `www.`? Curious what it's for?

**Review by daine — APPROVED** [review]
LGTM with minor comments/questions - happy to chat! 😄

File: app/utils/listItemsUtility.js (hunk 2 above)
Comment (dylanzheng94) [reply app/utils/listItemsUtility.js:49]: Screenshot 2023-03-06 at 7 43 18 PM

Removes 'www' from the source name for display purposes

File: app/utils/constants.js (hunk 1 above)
Comment (dylanzheng94) [reply app/utils/constants.js]: Agree, this logic has been removed and relocated to _resource.vue

File: app/utils/listItemsUtility.js (hunk 2 above)
Comment (daine) [reply app/utils/listItemsUtility.js:50]: Thanks! Appreciate the screenshot for added visualization 😍

**Review by daine — APPROVED** [review]

**Pull request merged**